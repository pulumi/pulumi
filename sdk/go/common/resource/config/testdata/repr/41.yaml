my:outer:
  inner: 0
