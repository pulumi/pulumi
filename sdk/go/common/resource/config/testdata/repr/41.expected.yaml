decrypt:
  my:outer: '{"inner":0}'
paths:
  my:outer:
    value:
      inner: 0
    string: '{"inner":0}'
    redacted: '{"inner":0}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAV/4ISAAEFaW5uZXIFaW50NjQEAgAA
    secure: false
    isObject: true
  my:outer.inner:
    value: "0"
    string: "0"
    redacted: "0"
    object: DhAABnN0cmluZwwDAAEw
    secure: false
    isObject: false
